lib/appengine/util/gcloud.rb

```diff
@@ -78,7 +78,7 @@ class << self
         #
         def binary_path
           unless defined? @binary_path
-            @binary_path = `which gcloud`.strip
+            Gem.win_platform? ? @binary_path = 'gcloud' : @binary_path = `which gcloud`.strip
```

Comment:
Thanks for the PR! I don't have easy access to Windows so this is really helpful.

This code (on linux/mac) is trying to determine if gcloud is actually installed (and available in the path). Is there a way to make this determination in Windows?

Reply:
I tried to do the same initially, but the path that i've from 'where' command is unusable(C:\\..\\.\\ style windows syntax, which ruby doesn't understand). So maybe I can use the 'where' command for checking whether the 'gcloud' is in path, if it is, i'll just write 'gcloud' to @binary_path? Let me just modify it like that and i will create another pull request.

Reply:
That looks good to me.